Mobile/ShopLens/Droid/IO/ShopLensPictureDirectoryCreator.cs

```diff
@@ -14,11 +14,7 @@ private void CreateDirectoryForPictures(File pictureDirectory)
         {
             if (!pictureDirectory.Exists())
             {
-                if (!pictureDirectory.Mkdirs())
```

Comment:
Here the method Mkdirs() returns false only when a said directory already exists, which is impossible, because we already checked that in the above line.

Reply:
Are you sure? I think it returned false for me when it didn't have access / the directory path was incorrect.

Reply:
It can return false only when there is no permission given to create new files (write), but since we're going to implement mandatory permission activation requirements for the user, it will never return false when there's no access given. If the user doesn't give the required permission, we won't let him use our app.